src/actions/oldInvitationsActions.js

```diff
@@ -142,7 +142,9 @@ export const fetchOldInviteNotificationsAction = () => {
     });
 
     await dispatch(updateConnectionsAction());
-    dispatch(getExistingChatsAction());
+    await dispatch(getExistingChatsAction());
```

Comment:
`return dispatch(getExistingChatsAction())` will await for promise to complete

Reply:
Changed and tested in 2a6fa98